### PR TITLE
Change update to upsert

### PR DESCRIPTION
**Pull request by Elkrival**
with the most recent logs it looks like the update method is not supported by document db, this pr updates the updateOne method to upsert.

### CODE REVIEW

File: server/controllers/api/assessmentDayDataController/index.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const AssessmentDayDataController = {
           ...sortedDayData.map((dayData) => dayData.day)
         )
         console.log('Participant data update')
-        await AssessmentDayDataModel.update(appDb, query, {
```
Comment (mikestone14) [inline server/controllers/api/assessmentDayDataController/index.js:70]: should we remove the `update` method on the `AssessmentDayDataModel` if we can't use it?

Comment (Elkrival) [reply server/controllers/api/assessmentDayDataController/index.js:70]: [46cd133](https://github.com/AMP-SCZ/dpdash/pull/753/commits/46cd133f33881e5a8089c4d30a2c7478471cc297)

File: server/models/AssessmentDayDataModel/index.js [hunk 2]
```diff
@@ -21,10 +21,6 @@ const AssessmentDayDataModel = {
     await db
       .collection(collections.assessmentDayData)
       .insertMany(assessmentDayData),
-  update: async (db, query, assessmentDayDataAttributes) =>
-    await db
-      .collection(collections.assessmentDayData)
-      .updateOne(query, { $set: assessmentDayDataAttributes }),
```
Comment (mikestone14) [inline server/models/AssessmentDayDataModel/index.js:27]: just checking - was this function referenced anywhere else in the codebase?

Comment (Elkrival) [reply server/models/AssessmentDayDataModel/index.js:27]: nope I scanned for other occurrences and there is just this one.

**Review by mikestone14 — APPROVED** [review]
assuming we've confirmed that the upsert method will work 👍

**Comment by Elkrival** [comment]
Tested locally

**Pull request merged**